src/aiida/cmdline/commands/cmd_code.py

```diff
@@ -260,9 +260,14 @@ def export(code, output_file, sort):
         if value is not None:
             code_data[key] = str(value)
 
+    if output_file is None:
+        output_file = f"{code_data['label']}.yml"
+
     with open(output_file, 'w', encoding='utf-8') as yfhandle:
         yaml.dump(code_data, yfhandle, sort_keys=sort)
 
+    echo.echo_success(f"Code<{code.pk}> {code.label} exported to file '{output_file}'.")
```

Comment:
Please stick to the convention of using backticks for variables
```suggestion
    echo.echo_success(f'Code<{code.pk}> {code.label} exported to file `{output_file}`.')
```

Reply:
Hm, I agree with being consistent here. For that reason, I used the same setup as for the `echo.echo_success` of `verdi computer export`. So I'll go ahead and modify it there, as well.